features/laser_features.py

```diff
@@ -155,13 +160,26 @@ class LaserState(traits.HasTraits):
     laser_power = traits.List([1.,], desc="Laser power (between 0 and 1) for each active laser")
     laser_pulse_width = traits.List([0.005,], desc="List of possible pulse widths in seconds")
     laser_poisson_mu = traits.Float(0.5, desc="Mean duration between laser stimulations (s)")
+    laser_min_isi = traits.Float(0.01, desc="Minimum duration between laser stimulation pulses (s)")
```

Comment:
just suggesting to add this comment here if true: 'This parameter set to 0.05 or above could avoid STDP when stimulating at two different sites'.